### ISSUE TITLE
Practical example: create handwritten digit samples to test a neural network for digit recognition.

### ISSUE THREAD

**Issue by drewkenjo** [Player-i/data-analysis-demo#1]
As a practical exercise: create your own handwritten digit samples to test a neural network trained on MNIST dataset

- Create 25 handwritten digit images, e.g. you can use your phone to take pics.
  - make sure to write using thick marker, and crop images
- Use Python libraries to process the images, convert them to monochrome, resize, and convert to the format that your keras model expects
- Share the example with the group, and what accuracy did you get from the model, trained on the original MNIST dataset
